### DISCUSSION TITLE
Privacy settings for user page

### DISCUSSION THREAD

**Discussion by RadiantEclipse2003** [RetroAchievements/RAWeb#1053] [Ideas]
Hi! Not everyone is comfortable with the user page publicly reporting your activity in Retro Achievements. It would be nice if it was added to the site a Privacy Settings for your user page. So users could choose whether they want their user page to be public, visible only to friends or private. This type of setting is very common in several services on the internet including PSN, Xbox Live and Steam so I believe it would be a great addition to Retro Achievements and for sure all users would like this addition.

**Comment by StingX2** [comment]
This would only come with an untracked status. We use activity reporting to track unsavory users.

**Reply by ScottFromDerby** [reply]
Do we want to make it so the user page just remains empty if you're another user or not logged in, but populated as normal if you're the local user, or developer or higher? Do we want to show badges or no?
User activity would be able to be found elsewhere as well, it could be tough to implement... but if the bulk is just to hide it from regular users or anons, I think there could be a use case. For instance if I somehow pop some cheevos between 9-5 Monday to Friday, my employer might not like it. However this definitely would have to come with untracked status too.

**Reply by pinguupinguu** [reply]
> Do we want to make it so the user page just remains empty if you're another user or not logged in, but populated as normal if you're the local user, or developer or higher? Do we want to show badges or no?
> User activity would be able to be found elsewhere as well, it could be tough to implement... but if the bulk is just to hide it from regular users or anons, I think there could be a use case. For instance if I somehow pop some cheevos between 9-5 Monday to Friday, my employer might not like it. However this definitely would have to come with untracked status too. 

I dont think this is a bad move, but this kind of issue should not be implemented until we actually have a permissions matrix with moderation permissions

**Comment by Jamiras** [comment]
See also #971
